integration-test/src/main/java/com/sequenceiq/it/cloudbreak/AzureTemplateCreationTest.java

```diff
@@ -34,7 +34,7 @@ public void testAzureTemplateCreation(@Optional("it-azure-template") String azur
         TemplateRequest templateRequest = new TemplateRequest();
         templateRequest.setName(azureTemplateName);
         templateRequest.setDescription("AZURE template for integration testing");
```

Comment:
maybe the description should be also 'AZURE_RM template for integration testing'